app/views/home/_nav.html.haml

```diff
@@ -0,0 +1,21 @@
+%div.fixed
+  %nav.top-bar{ "data-topbar" => true, "role" => "navigation" }
```

Comment:
Just for future reference, apparently the correct way to do any `data-*` attributes in haml is `{ data: { topbar: true }`. Don't need to change it though.

http://haml.info/docs/yardoc/file.REFERENCE.html#html5_custom_data_attributes